### PR TITLE
sys/test: first test script, tests boot

### PR DESCRIPTION
**Pull request by sevki**
this is a first pass at the boot test. 

Signed-off-by: Sevki s@sevki.org

### CODE REVIEW

File: sys/test/BUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "dump,file=/tmp/vm0.pcap",
+    ],
+    redir=[
+        "tcp:9999::9",
+        "tcp:17010::17010",
+        "tcp:17013::17013",
+    ],
+    append="service=cpu nobootprompt=rc maxcores=1024 fs=10.0.2.2 auth=10.0.2.2 nvram=/boot/nvram nvrlen=512 nvroff=0 -user=harvey",
+    cmds=[
+        "/boot/ls",
```
Comment (sevki) [inline sys/test/BUILD]: right :)

**Comment by sevki** [comment]
yes sir

**Comment by sevki** [comment]
inside the harvey directory 

```
go get -u sevki.org/build/cmd/build
build -v //sys/test:boot
```

File: BUILD [hunk 2]
```diff
@@ -94,11 +94,15 @@ kernel(
 		"amd64/bin/date": "bin/date",
```
Comment (rminnich) [inline BUILD]: What I would prefer, given how smart bldy is, is to just have the kernel depend on //sys/src/cmd:cmd

File: sys/src/9/amd64/BUILD [hunk 3]
```diff
@@ -266,14 +267,23 @@ cc_binary(
 		"-T",
 	]
 )
+move(
+	name="factotum",
+	deps=[
+		"//sys/src/cmd/auth/factotum:factotum",
+	],
+	installs={
+		"bin/auth/factotum": "bin/factotum",
```
Comment (rminnich) [inline sys/src/9/amd64/BUILD]: er, what? Why?

File: BUILD (hunk 2 above)
Comment (rminnich) [inline BUILD]: where are these things being moved? Maybe I am just not understanding what you're doing here. I don't get it.

**Pull request merged**